### DISCUSSION TITLE
Do you accept PRs for modernizing System.Web.HttpUtility?

### DISCUSSION THREAD

**Discussion by TrayanZapryanov** [dotnet/runtime#102546] [Ideas] [answered]
We are using [System.Web.HttpUtility](https://github.com/dotnet/runtime/tree/main/src/libraries/System.Web.HttpUtility) and when checked it's code I saw several byte/char[] allocations that can be optimized by using stackalloc.

If you accept PRs, I can work a bit in this solution.

**Comment by huoyaoyuan** [comment, marked as answer]
There is merged community PR within this year, so I think it's open for contribution.

If you want to do improvement, consider including benchmark to show its value.

**Reply by TrayanZapryanov** [reply]
Thank you.
Once I can see something useful will submit a small PR to see if it is accepted.

**Reply by MihaZupan** [reply]
Thank you for looking into improving things.
Provided that the change is well covered by unit tests, we're open to accepting performance improvements.
